telco-examples/edge-clusters/dhcp-less/edge-metallb-multi-node/bmh-node1-example.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: node1-example-credentials
+type: Opaque
+data:
+  username: ${BMC_NODE1_USERNAME}
+  password: ${BMC_NODE1_PASSWORD}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: controlplane-0-networkdata
+type: Opaque
+stringData:
+  networkData: |
+    interfaces:
+    - name: ${CONTROLPLANE1_INTERFACE}
+      type: ethernet
+      state: up
+      mac-address: "${CONTROLPLANE1_MAC}"
+      ipv4:
+        address:
+        - ip:  "${CONTROLPLANE1_IP}"
+          prefix-length: "${CONTROLPLANE1_PREFIX}"
+        enabled: true
+        dhcp: false
+    dns-resolver:
+      config:
+        server:
+        - "${DNS_SERVER}"
+    routes:
+      config:
+      - destination: 0.0.0.0/0
+        next-hop-address: "${CONTROLPLANE1_GATEWAY}"
+        next-hop-interface: ${CONTROLPLANE1_INTERFACE}
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: node1-example
+  labels:
+    cluster-role: control-plane
+spec:
+  online: true
+  bootMACAddress: ${BMC_NODE1_MAC}
+  bmc:
+    address: ${BMC_NODE1_ADDRESS}
+    disableCertificateVerification: true
+    credentialsName: node1-example-credentials
```

Comment:
you need to add the preprovisioningNetworkDataName option to point to the secret there

Reply:
for some reason I didn't copy the last line from the PW manifest. 
Thanks